src/components/classifier/Tutorial.js

```diff
@@ -23,23 +23,27 @@ export class Tutorial extends Component {
     super(props)
     this.state = {
       step: 0,
-      sizedHeight: 0,
-      sizedWidth: 0,
     }
   }
 
   render() {
     const steps = this.props.tutorial.steps
-    const step = steps[this.state.step]
     const totalSteps = length(steps)
-
-    const mediaResource = (step.media ? this.props.tutorial.mediaResources[step.media] : null )
-    const mediaImage = (mediaResource !== null ? <SizedImage source={{ uri: mediaResource.src }} /> : null)
+    
+    const tutorialStep = steps.map((step, index) => {
+      return (
+        <TutorialStep
+          key={`${index}`}
```

Comment:
Might be safer here to make the key `TUTORIAL_STEP_${index}` or something similar to avoid the key from interfering with any other z-indexed keys in the view.